### ISSUE TITLE
fail2ban doesnt ban attempts with invalid user

### ISSUE THREAD

**Issue by rfr** [fail2ban/fail2ban#2685]
### Environment:
v0.10.2 on ubuntu 18.04.1 LTS

### The issue:

Hey guys! Not sure whether it's an intended behaviour. Fail2ban doesn't ban attempts to login with an invalid user. Couldn't find anything related in the filters. I get lots of those in my logs, imho it would  be great to ban those addresses too. Examples:

```
Apr  8 20:06:08 ubuntu-16gb-nbg1-1 sshd[30827]: Invalid user ubuntu from 106.12.14.130 port 34362
Apr  8 20:06:09 ubuntu-16gb-nbg1-1 sshd[30827]: Received disconnect from 106.12.14.130 port 34362:11: Bye Bye [preauth]
Apr  8 20:06:09 ubuntu-16gb-nbg1-1 sshd[30827]: Disconnected from invalid user ubuntu 106.12.14.130 port 34362 [preauth]
```

**Comment by sebres** [comment]
This was fixed in 8028d3940d0173582adede443f8692b510465917...
See #2090 for details (previously no `port ...` was expected here).

**Comment by rfr** [comment]
@sebres Hi Sergey, thanks! I now see that I have an older version. Can you point me to an instruction on how to install the newest version like 0.11.1?
